server/graphql/v2/object/PaymentMethod.js

```diff
@@ -62,13 +62,42 @@ export const PaymentMethod = new GraphQLObjectType({
             return pick(paymentMethod.data, ['email']);
           }
 
+          // Return HostCollectiveId for prepaid
+          if (paymentMethod.type === 'prepaid') {
+            return pick(paymentMethod.data, ['HostCollectiveId']);
+          }
+
           const data = paymentMethod.data;
           // white list fields to send back; removes fields like CustomerIdForHost
           const dataSubset = pick(data, ['fullName', 'expMonth', 'expYear', 'brand', 'country', 'last4']);
 
           return dataSubset;
         },
       },
+      limitedToCollectiveIds: {
```

Comment:
I just reviewed this one and I'm wondering if we need to keep supporting it. I just checked and the API support for this property is flawed, proving that it's not much used. Decision tomorrow.

In case we decided to keep it, similar to my comment below in `limitedToHostCollectiveIds`, it should be exposed as a list of `Account`s.

Reply:
As discussed, I'm removing this field in https://github.com/opencollective/opencollective-api/pull/4361